service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -99,12 +104,62 @@ void shouldSealPDFAndAddToSealedCMOsListWhenJudgeApprovesOrder() throws Exceptio
             .status(APPROVED)
             .build();
 
+        assertThat(State.ISSUE_RESOLUTION).isNotEqualTo(responseData.getState());
         assertThat(responseData.getDraftUploadedCMOs()).isEmpty();
         assertThat(responseData.getSealedCMOs())
             .extracting(Element::getValue)
             .containsExactly(expectedSealedCmo);
     }
 
+    @Test
+    void shouldUpdateStateToIssueResolutionWhenNextHearingIssueResolutionAndCmoDecisionIsSendToAllParties()
+        throws Exception {
+        DocumentReference convertedDocument = testDocumentReference();
+        DocumentReference sealedDocument = testDocumentReference();
+
+        given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
+        given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);
+        given(featureToggleService.isNewCaseStateModelEnabled()).willReturn(true);
+
+        UUID cmoId = UUID.randomUUID();
+        CaseData caseData = buildCaseData(cmoId, SEND_TO_ALL_PARTIES);
+        CaseData responseData = extractCaseData(postAboutToSubmitEvent(caseData));
+
+        assertThat(State.ISSUE_RESOLUTION).isEqualTo(responseData.getState());
+    }
+
+    @Test
+    void shouldNotUpdateStateToIssueResolutionWhenFeatureToggledOff() throws Exception {
+        DocumentReference convertedDocument = testDocumentReference();
+        DocumentReference sealedDocument = testDocumentReference();
+
+        given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
+        given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);
+        given(featureToggleService.isNewCaseStateModelEnabled()).willReturn(false);
+
+        UUID cmoId = UUID.randomUUID();
+        CaseData caseData = buildCaseData(cmoId, SEND_TO_ALL_PARTIES);
+        CaseData responseData = extractCaseData(postAboutToSubmitEvent(caseData));
+
+        assertThat(State.ISSUE_RESOLUTION).isNotEqualTo(responseData.getState());
+    }
+
+    @Test
```

Comment:
Do we need a test to cover shouldNotUpdateStateToIssueResolutionWhenNextHearingIsNotIssueResolution? and shouldNotUpdateStateToIssueResolutionWhenNoNextHearingPresent? Let me know what you think :)

Reply:
Discussed, added an additional test to test scenario next hearing is not Issue resolution.